pkgs/unified_analytics/lib/src/enums.dart

```diff
@@ -219,4 +219,16 @@ enum DevicePlatform {
 
   final String label;
   const DevicePlatform(this.label);
+
+  /// Returns the [DevicePlatform] that matches the provided [os].
+  ///
+  /// Where possible, the value for [os] should be derived from
+  /// 'Platform.operatingSystem' from dart:io.
+  static DevicePlatform fromOs(String os) {
```

Comment:
Would it be better to pass the `io.Platform` class instead? Just incase someone passes the wrong string and it always defaults to windows

Reply:
Is it okay to have a dart:io in this library?

Reply:
Ah good call! In that case, can you add to the dart doc what the expected values for the switch expression should be, `macos`, `linux`, or `windows`?